unity/Runtime/DissonityApi.cs

```diff
@@ -102,6 +103,19 @@ public static string InstanceId
             }
         }
         
+        /// <summary>
+        /// Your client access token
+        /// </summary>
+        public static string? AccessToken
+        {
+            get
+            {
+                if (!_ready) throw new InvalidOperationException("You can't access this property before waiting for Api.Initialize");
+
+                return _accessToken;
```

Comment:
Since authentication is required for an activity to work, this type doesn't need to be nullable
```suggestion
        public static string AccessToken
        {
            get
            {
                if (!_ready) throw new InvalidOperationException("You can't access this property before waiting for Api.Initialize");

                return _accessToken!;
```